src/site/content/en/learn/css/backgrounds/index.md

```diff
@@ -0,0 +1,456 @@
+
+## Background Color
+
+One of the simplest effects you can apply to a background layer is set a [valid CSS color](). The initial value of `background-color` is `transparent`, which allows the contents of a parent to be seen through a CSS box.
+
+A color set on a background layer sits behind other things painted on a CSS box. This includes borders and outlines (editor’s note: Will deliberately set background-origin to border-box). The demo below demonstrates this.
+
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'XWgRRMQ',
+  tab: 'css,result'
+} %}
+
+A `background-color` is also guaranteed to sit behind other layers you can create with background-related CSS properties such as a background-image. More on that next.
+
+## Background Image
+
+On top of the `background-color` is a background image, added with the `background-image` property. A `background-image` accepts the following:
+
+* A URL or data URLI using the `url` CSS function.
+* An image dynamically created by a gradient CSS function.
+
+### Setting a background-image with the `url` CSS function.
+
+The `url()` CSS  function enables CSS authors to reference a resource as an absolute URL, relative URL, or embed a resource itself in a CSS document by referencing the resource as a  data URI.
+
+The following demo demonstrates this:
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'JjJNNro',
+  tab: 'css,result'
+} %}
+
+{% Aside %}
+What’s the difference between  absolute URls, relative URLs, and data URIs?
+
+URL is short for uniform resource locator a standardized way to find resources on the web.
+
+*Absolute URLs* are URLs with all the information needed for a browser to fetch a resource at a particular location on the web regardless where the request originated from.
+
+
+```
+#someElement {
+  background-image: url('https://images.unsplash.com/photo-1536707076730-0615b7f5b49a');
+}
+```
+
+Whether in this stylesheet or referenced directly in a browser address bar, `https://images.unsplash.com/photo-1536707076730-0615b7f5b49a` is a URL that can be used to consistently & securely fetch a particular image from images.unsplash.com.
+
+*Relative URLs* are URLs that instruct the browser to attempt to fetch a resource relative to the file the URL was embedded in.
+
+```
+#anotherElement {
+  background-image: url(‘.../some-relative-path/image.png’)
+}
+```
+
+The `.../some-relative-path/image.png` URL within the stylesheet enables an image to be reliably fetched assuming the location of both the stylesheet & image does not change.
+
+**Data URIs** are a means of describing a resource as bytes of data embedded where it is described; when used by the URL function, you're effectively “inling” the file.
+
+
+A data URL represents an image as a base64 encoded string. This leads to often a very long string necessary to represent the image accurately:
+
+```
+#demoBox {
```

Comment:
Can you actually cut this off rather than including it? Just add some "..."'s after 80-100 characters.